registry/mainnet/interchain/squid.tokenlist.json

```diff
@@ -2809,6 +2809,118 @@
           "tokenManagerType": "mintBurn"
         }
       ]
+    },
+    "0x188ca79eaa83cc4d76b71eda77b843636ca03164d1c3a986b545b0695681ec9a": {
+      "tokenId": "0x188ca79eaa83cc4d76b71eda77b843636ca03164d1c3a986b545b0695681ec9a",
+      "deployer": "0x20E0CB3f865661daeE1Cc5D48E7d1440fF0A45Cc",
+      "originalMinter": null,
+      "prettySymbol": "KATA",
+      "decimals": 18,
+      "originAxelarChainId": "ethereum",
+      "tokenType": "canonical",
+      "deploySalt": "0x",
```

Comment:
Salt should be filled with the value of the salt you used when you were deploying the tokens from the portal. Should be in the event logs in case you cannot find it.